gradio/routes.py

```diff
@@ -20,10 +21,9 @@
 import threading
 import time
 import traceback
-from asyncio import TimeoutError as AsyncTimeOutError
-from collections import defaultdict
 from pathlib import Path
-from typing import TYPE_CHECKING, Any, Dict, List, Optional, Type
+from queue import Empty as EmptyQueue
+from typing import TYPE_CHECKING, Any, AsyncIterator, Dict, List, Optional, Type
```

Comment:
reload mode is connecting via websocket.
I think we can refactor that to use SSE as well?
Would allow us to get rid of websockets dependency in the requirements file.
We can also get rid of `ws_login_check` right now though.